doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -102,13 +102,13 @@ for all processes of all extant cluster replicas, as a percentage of the total r
 At this time, we do not make any guarantees about the exactness or freshness of these numbers.
 
 <!-- RELATION_SPEC mz_internal.mz_cluster_replica_utilization -->
-| Field            | Type                 | Meaning                                                    |
-|------------------|----------------------|------------------------------------------------------------|
-| `replica_id`     | [`text`]             | The ID of a cluster replica.                               |
-| `process_id`     | [`uint8`]            | An identifier of a compute process within a replica.       |
-| `cpu_percent`    | [`double precision`] | Approximate CPU usage, in percent of the total allocation. |
-| `memory_percent` | [`double precision`] | Approximate RAM usage, in percent of the total allocation. |
-| `disk_percent`   | [`double precision`] | Currently null. Reserved for later use.                    |
+| Field            | Type                 | Meaning                                                                            |
+|------------------|----------------------|------------------------------------------------------------------------------------|
+| `replica_id`     | [`text`]             | The ID of a cluster replica.                                                       |
+| `process_id`     | [`uint8`]            | An identifier of a compute process within a replica.                               |
+| `cpu_percent`    | [`double precision`] | Approximate CPU usage, in percent of the total allocation.                         |
+| `memory_percent` | [`double precision`] | Approximate RAM usage, in percent of the total allocation.                         |
+| `disk_percent`   | [`double precision`] | Approximate disk usage in % of the attached disk, if there is one. null otherwise. |
```

Comment:
```suggestion
| `disk_percent`   | [`double precision`] | Approximate disk usage in % of the attached disk, if there is one. `NULL` otherwise. |
```